src/components/Dao/DaoSchemesPage.tsx

```diff
@@ -98,13 +140,21 @@ class DaoSchemesPage extends React.Component<IProps, null> {
   }
 }
 
-export default withSubscription({
+const SubscribedDaoSchemesPage = withSubscription({
   wrappedComponent: DaoSchemesPage,
   loadingComponent: <Loading/>,
   errorComponent: (props) => <span>{props.error.message}</span>,
   checkForUpdate: [],
   createObservable: (props: IExternalProps) => {
+    const arc = getArc();
     const dao = props.daoState.dao;
-    return dao.schemes({ where: { isRegistered: true } }, { fetchAllData: true, subscribe: true });
+
+    return combineLatest(
+      dao.schemes({ where: { isRegistered: true } }, { fetchAllData: true, subscribe: true }),
+      // Find the SchemeManager scheme if this dao has one
+      Scheme.search(arc, {where: { dao: dao.id, name: "SchemeRegistrar" }}).pipe(mergeMap((scheme: Array<Scheme | CompetitionScheme>): Observable<ISchemeState> => scheme[0] ? scheme[0].state() : of(null)))
```

Comment:
why is CompetitionScheme needed here?  We're looking for SchemeRegistrar.

can we document that in the code?

Reply:
thats the type returned by Scheme.search so thats why we need to use it